README.md

```diff
@@ -72,11 +73,12 @@ Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of conduc
 
 ## Versioning
 
-We employ [auto-changelog](https://www.npmjs.com/package/auto-changelog) to manage the [CHANGELOG.md](CHANGELOG.md). For the versions available, see the [tags on this repository](https://github.com/GoodwayGroup/gwvault/tags).
+We employ [git-chglog](https://github.com/git-chglog/git-chglog) to manage the [CHANGELOG.md](CHANGELOG.md). For the versions available, see the [tags on this repository](https://github.com/GoodwayGroup/gwvault/tags).
 
 ## Authors
 
 * **Derek Smith** - [@clok](https://github.com/clok)
+* **Paulo Black** - [@clok](https://github.com/paulojblack)
```

Comment:
https://media.giphy.com/media/5xtDarmwsuR9sDRObyU/giphy.gif

Reply:
Whoops! I messed up the alias. Fixing now.